ansible_collections/arista/avd/examples/l2ls-fabric/README.md

```diff
@@ -289,56 +289,57 @@ Our example used an external L3/FW Device to route between subnets. This is very
 
     The spine type has been changed to **l3spine**.
 
-![Figure: 3](images/L2LS_Spine_routing.svg)
+![Figure: 3](images/L2LS_spine_routing.svg)
 
 The following group_vars need updating to enable L3 routing on the spines.
 
-- DC1_SPINES.yml
-- DC1_FABRIC.yml
-- DC1_NETWORK_SERVICES.yml
+- DC1_SPINES/spines.yml
+- DC1_NETWORK_SERVICES/services.yml
```

Comment:
`services.yml` is the only file in DC1_NETWORK_SERVICES. Please do not create dir/file structure for that, simply use DC1_NETWORK_SERVICES.yml. Same applies to DC1_SPINES.yml and others. In my opinion this can confuse people. We only need a directory when we want to split the data into multiple files. Good example is connected endpoints, however to show how that works and the value we need some "scale". Therefore I'd suggest to keep using <group-name>.yml everywhere and craft a dedicated example for scaled provisioning use cases later.